onnx/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 # isort:skip_file
 import os
```

Comment:
Prefer importing modules only
——
I would avoid importing a class or a function from a module.

Import only the module instead to keep the name space clean and provide readers with more context on where its members come from. For example, readers don’t need to backtrack to see `sleep` is a function from `time`, as opposed to a function defined in the file. https://google.github.io/styleguide/pyguide.html#22-imports